src/game.py

```diff
@@ -1,48 +1,76 @@
 import math
-from pawn import Pawn, translate_x, translate_y
+from pawn import Orientation, Pawn, translate_x, translate_y
 from exception import QuoridorException, OutOfBoardException, UnknownActionException
 from functools import reduce
 from action import Action
 import console
 from board import *
 
+from enum import IntEnum
+
+
+class State(IntEnum):
+    RUNNING = 0,
+    VICTORY = 1,
+    QUIT = 2,
+
 
 def init_game():
     center = math.floor(BASE_LINE_SIZE / 2)
-    return Pawn(0, center)
+    return [Pawn(0, center, Orientation.WEST), Pawn(BASE_LINE_SIZE - 1, center, Orientation.EAST)]
 
 
-def progress(pawn):
-    new_pawn = pawn
+def progress(pawns):
+    new_pawns = pawns
     new_fences = build()
-    quit = False
-    victory = False
+    player_turn = 1
+    state = State.RUNNING
     console.clear()
-    console.prompt("** Welcome to PyQuoridor **\n  Press Enter to start ...")
-    while not(quit or victory):
-        display(new_pawn, new_fences)
-        action = console.prompt_action()
+    console.prompt("\n *** Welcome to PyQuoridor ***\n  Press Enter to start ...")
+    while state == State.RUNNING:
+        display(new_pawns, new_fences)
+        action = console.prompt_action(player_turn)
         if action == Action.EXIT:
-            quit = True
+            state = State.QUIT
         else:
             try:
-                new_pawn = act(action, new_pawn, new_fences)
+                new_pawn = act(action, new_pawns[player_turn - 1], new_fences)
+                new_pawns = deepcopy(new_pawns)
```

Comment:
I'd recommend to deepcopy `pawns` at start, and then update new_pawns as you need